### PR TITLE
Fixed code for selling nft on getgems

### PR DESCRIPTION
**Pull request by vityooook**
Update GetGems NFT sale code for compatibility with v3r3 smart contract

This pull request updates the NFT sale code to work with the latest GetGems smart contract (v3r3). The previous version (v3r2) is no longer compatible.

Changes:

• Updated code for NFT sale functionality on all three supported languages (Python, JS, and Go) to work with the new v3r3 smart contract.
• Tested code thoroughly for Python and JS. Go code has not been tested, but I think it works well.
• Minor code clean-up and refactoring in other areas of the repository.

Benefits:

• Improved compatibility with the latest GetGems smart contract.
• Ensures smooth operation of the NFT sale process for users.

Link: https://docs.ton.org/develop/dapps/ton-connect/message-builders#nft-sale-getgems

### CODE REVIEW

**Comment by reveloper** [comment]
@vityooook, thank you for this super clean PR! We appreciate your hard work.

File: docs/develop/dapps/ton-connect/message-builders.mdx [hunk 1]
```diff
@@ -701,7 +701,7 @@ import { beginCell, toNano } from '@ton/ton'
         .storeAddress(Address.parse(NEW_OWNER_WALLET)) // new_owner:MsgAddress
         .storeAddress(Address.parse(Wallet_DST))       // response_destination:MsgAddress
         .storeUint(0, 1)                         // custom_payload:(Maybe ^Cell)
-        .storeCoins(toNano(0.000000001).toString())       // forward_amount:(VarUInteger 16)
+        .storeCoins(toNano("0.01"))              // forward_amount:(VarUInteger 16)
```
Comment (reveloper) [inline docs/develop/dapps/ton-connect/message-builders.mdx:704]: We need to keep 1 nanoTon here, as we want to send a message with the minimal amount.

File: docs/develop/dapps/ton-connect/message-builders.mdx [hunk 2]
```diff
@@ -598,7 +598,7 @@ The `body` for Jetton Burn([TEP-74](https://github.com/ton-blockchain/TEPs/blob/
     const body = beginCell()
         .storeUint(0x595f07bc, 32)                // jetton burn op code
         .storeUint(0, 64)                         // query_id:uint64
-        .storeCoins(1000000)                      // amount:(VarUInteger 16) -  Jetton amount in decimal
+        .storeCoins(toNano("0.001"))                      // amount:(VarUInteger 16) -  Jetton amount in decimal
```
Comment (reveloper) [inline docs/develop/dapps/ton-connect/message-builders.mdx:601]: Let's add clarification, that decimal size individually specified for every jetton by its master contract

File: docs/develop/dapps/ton-connect/message-builders.mdx [hunk 3]
```diff
@@ -440,11 +440,11 @@ The `messageBody` for Jetton Transfer([TEP-74](https://github.com/ton-blockchain
     const body = beginCell()
         .storeUint(0xf8a7ea5, 32) // opcode for jetton transfer
         .storeUint(0, 64) // query id
-        .storeCoins(toNano(5)) // jetton amount, amount * 10^9
+        .storeCoins(toNano("5")) // jetton amount, amount * 10^9
```
Comment (reveloper) [inline docs/develop/dapps/ton-connect/message-builders.mdx:443]: Let's add clarification, that decimal size individually specified for every jetton by its master contract

Comment (vityooook) [reply docs/develop/dapps/ton-connect/message-builders.mdx:443]: ok, I'll fix everything soon

**Comment by vityooook** [comment]
@reveloper Alex, I have supplemented the documentation at your request.

**Comment by reveloper** [comment]
@vityooook, thank you!

**Pull request merged**